tests/test_cli.py

```diff
@@ -116,6 +116,20 @@ def test_render_output_build_path(testing_workdir, testing_metadata, capfd, capl
     assert error == ""
 
 
+def test_render_output_build_path_and_file(testing_workdir, testing_metadata, capfd, caplog):
+    api.output_yaml(testing_metadata, 'meta.yaml')
+    rendered_filename = 'out.yaml'
+    args = ['--output', '--file', rendered_filename, os.path.join(testing_workdir)]
```

Comment:
is that right?
``os.path.join(testing_workdir)``

Reply:
>is that right?
>`os.path.join(testing_workdir)`

Good point.  I created this test by copying/modifying the test above it.  I have no idea why that test is calling `join()` with only a single argument.  I agree with you -- it seems like a mistake.  Now I've pushed a commit to fix my test and the two others in the file making the same mistake.